File: DotNet/Framework/AutomationFramework/Tests.UITests/DashboardPageTests/DashboardHeaderTest.cs
using NUnit.Framework;
using Pages;

namespace Tests.UITests.DashboardPageTests
{
    [TestFixture]
    public class DashboardHeaderTest : BaseTestFixtures
    {
        DashboardPage Dashboard_Page;

        [SetUp] public void SetUp() 
        {
            Dashboard_Page = LoginUser();
        }

        [Test]
        public void HeaderTest() 
        {
            Dashboard_Page.GetHeaderText(out string header);

            Assert.That(header, Is.EqualTo("Dashboard"));
        }
    }
}

File: Java/AutomationFramework/src/com/app/UITests/DashboardPageTests/DashboardHeaderTest.java
package com.app.UITests.DashboardPageTests;

import com.app.pages.DashboardPage;
import org.testng.Assert;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class DashboardHeaderTest extends BaseDashboardTest{
    DashboardPage Dashboard_Page;

    @BeforeMethod
    public void SetUp()
    {
        Dashboard_Page = LoginUser();
    }

    @Test
    public void HeaderTest()
    {
        String header = Dashboard_Page.GetHeaderText();

        Assert.assertEquals(header, "Dashboard");
    }
}
